agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -1766,6 +1767,17 @@ bool backhaul_manager::handle_1905_discovery_query(ieee1905_1::CmduMessageRx &cm
         return false;
     }
 
+    auto tlvDeviceInformation = cmdu_tx.addClass<ieee1905_1::tlvDeviceInformation>();
+    if (!tlvDeviceInformation) {
+        LOG(ERROR) << "addClass ieee1905_1::tlvDeviceInformation failed, mid=" << std::hex
+                   << (int)mid;
+        return false;
+    }
+    tlvDeviceInformation->mac() = network_utils::mac_from_string(bridge_info.mac);
+
+    //TODO: set number of local interfaces.
+    //TODO: fill info of each of the local interfaces, according to IEEE_1905 section 6.4.5
```

Comment:
Please add a link to the github issue - https://github.com/prplfoundation/prplMesh/issues/300